packages/html-reporter/src/reportView.css

```diff
@@ -33,8 +33,57 @@ body {
   border-top: 1px solid var(--color-border-default);
 }
 
+.htmlreport {
+  gap: 24px;
```

Comment:
non-existent!!

Reply:
It actually does something in flex context! Confirmed via toggling it in DevTools, and [MDN agrees](https://developer.mozilla.org/en-US/docs/Web/CSS/gap?msclkid=21de9013a66c11ecbcbb669f896b9dfe):

> <html>
<body>
<!--StartFragment-->

Applies to | multi-column elements, flex containers, grid containers
-- | --


<!--EndFragment-->
</body>
</html>

Reply:
TIL!